plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -48,7 +48,7 @@ library
     , extra
     , filepath
     , ghc-exactprint        >=0.6.3.4
-    , ghcide                >=1.2     && <1.4
+    , ghcide                ^>=1.4
```

Comment:
This changes because hlint plugin probably won't work with ghcide <= 1.3 after #1862